src/status_im/communities/core.cljs

```diff
@@ -784,3 +778,43 @@
                                 :community-id community-id
                                 :public-key   public-key
                                 :role-id      role-id})]})
+
+(rf/defn handle-fetched-collapsed-community-categories
+  {:events [:communities/fetched-collapsed-community-categories]}
```

Comment:
is this the right naming for the event?

Reply:
The convention is different @cammellos, although unfortunately it's not yet on the guidelines. We did have discussions that settled on some patterns, but it's not written in stone.

This event should have the `-success` suffix added from the event that actually triggered this one. This is very common in re-frame apps, particularly when the event is a response from an HTTP/RPC/etc call.

The names of events are usually in the present tense too, just like when we name functions. The name states what the function (or event) does, not what happened before it was called.

So to summarize, I would name the events:

- `:communities/fetched-collapsed-community-categories` becomes `:communities/fetch-collapsed-categories`.
- `:communities/toggled-collapse-category` becomes `:communities/toggle-collapsed-category` (small typo in the `collapsed` word.
- `:communities/toggled-collapse-category` becomes `:communities/toggle-collapsed-category-success`.
- Remove the word `handle-` from the event var. So `handle-toggle-collapse-category` becomes `toggle-collapsed-category` and `handle-toggled-collapse-category` becomes `toggle-collapsed-category-success`.

Reply:
on-fetch-collapsed-community-categories-sucess , dunno :)